pandas/core/frame.py

```diff
@@ -6891,7 +6891,7 @@ def cov(self, min_periods=None):
 
         return self._constructor(baseCov, index=idx, columns=cols)
 
-    def corrwith(self, other, axis=0, drop=False):
+    def corrwith(self, other, axis=0, method='pearson', drop=False):
```

Comment:
Hmm...it would be safest to add it to the end of the signature, as this modification as it stands is not backwards compatible.  Alternatively, you could issue a `DeprecationWarning` if a boolean is passed to `method` and re-assign it to `drop`.